packages/mobile/src/identity/contactMapping.ts

```diff
@@ -165,7 +168,7 @@ export function* fetchAddressesAndValidateSaga({ e164Number }: FetchAddressesAnd
     // Clear existing entries for those numbers so our mapping consumers know new status is pending.
     yield put(updateE164PhoneNumberAddresses({ [e164Number]: undefined }, {}))
 
-    const addresses: string[] | null = yield call(getAddresses, e164Number)
+    const addresses: string[] = yield call(getAddresses, e164Number)
```

Comment:
Thanks for addressing this but now you also need to adjust line 176 below 
`if (!address...`

Reply:
Yikes, can't believe I missed that. Good catch...thanks!